lib/ukalloc/alloc.c

```diff
@@ -65,9 +69,12 @@ int uk_alloc_register(struct uk_alloc *a)
 }
 
 struct metadata_ifpages {
+#ifdef CONFIG_ARM64_FEAT_MTE
+	__sz		size;
+#endif /* CONFIG_ARM64_FEAT_MTE */
 	unsigned long	num_pages;
 	void		*base;
-};
+} __attribute__ ((aligned(16)));
```

Comment:
I think there is a `__aligned` somewhere in unikraft that does this, you can use that

Reply:
Fixed, thanks 👍🏼